profiling/src/exception.rs

```diff
@@ -34,7 +34,9 @@ impl ExceptionProfilingStats {
     }
 
     fn track_exception(&mut self, name: String) {
-        if self.next_sample.checked_sub(1).is_none() {
+        let next_sample = self.next_sample.checked_sub(1);
+        if next_sample.is_some() {
+            self.next_sample = next_sample.unwrap();
```

Comment:
Obvious opportunity to not have to unwrap (didn't test, browser-written code):

```suggestion
        if let Some(next_sample_ = self.next_sample.checked_sub(1) {
            self.next_sample = next_sample;
```